app/code/Magento/Backend/Model/Search/ItemsFactory.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Backend\Model\Search;
+
+use Magento\Backend\Api\Search\ItemsInterface;
+use Magento\Framework\ObjectManagerInterface;
+
+class ItemsFactory
```

Comment:
Please consider using `ItemFactory` name for this class to be consistent with other Factories.